src/Life/Configuration.hs

```diff
@@ -63,10 +65,11 @@ instance Semigroup LifeConfiguration where
     life1 <> life2 = LifeConfiguration
         { lifeConfigurationFiles       = life1^.files <> life2^.files
         , lifeConfigurationDirectories = life1^.directories <> life2^.directories
+        , lifeConfigurationBranch      = life1^.branch <> life2^.branch
         }
 
 instance Monoid LifeConfiguration where
-    mempty  = LifeConfiguration mempty mempty
+    mempty  = LifeConfiguration mempty mempty (Last $ Just master)
```

Comment:
Unfortunately, having `master` as a default inside `mempty` violates laws for `Monoid` typeclass :disappointed: 

If you have:

```haskell
lc = LifeConfiguration mempty mempty smth
```

Then according to monoid neutral law you should have:

```haskell
lc1 <> mempty = lc1
```

but you will have `mempty` instead. I guess proper way of doing this is to somehow check value of `branch` field after parsing the file.

Maybe something like `Phase` approach from `summoner` can be used here:

* https://github.com/kowainik/summoner/blob/cf3abb978a4bfa9f74755465135bdbf272c08947/src/Summoner/Config.hs#L88-L97

No sure though about proper solution...

Reply:
I dropped `master` from Monoid instance and added `defaultLifeConfig` constant.